rskj-core/src/main/java/co/rsk/config/mining/HttpGetStableMinGasSystemConfig.java

```diff
@@ -0,0 +1,39 @@
+package co.rsk.config.mining;
+
+import com.typesafe.config.Config;
+
+public class HttpGetStableMinGasSystemConfig {
+    public static final String HTTP_GET_STABLE_GAS_PRICE_CONFIG_PATH = "miner.httpGet";
+    private static final String URL_PROPERTY = "url";
+    private static final String JSON_PATH_PROPERTY = "jsonPath";
+    private static final String TIMEOUT_PROPERTY = "timeout";
+    private static final String API_KEY_PROPERTY = "apiKey";
+
+    private final String url;
+    private final String jsonPath;
+    private final int timeout;
+    private final String apiKey;
+
+    public HttpGetStableMinGasSystemConfig(Config config) {
+        this.url = config.getString(URL_PROPERTY);
+        this.jsonPath = config.getString(JSON_PATH_PROPERTY);
+        this.timeout = config.getInt(TIMEOUT_PROPERTY);
+        this.apiKey = config.getString(API_KEY_PROPERTY);
+    }
```

Comment:
If you agree with the suggestion I have added on the Jira ticket, I would say that this object here could receive the map with the parameters and then initialise the properties according it. If it's missing some property, an exception could be thrown.

Reply:
As mentioned I do not see the advantage of having a map instead of this.